zigate/clusters.py

```diff
@@ -311,6 +327,18 @@ def __init__(self, endpoint=None):
                                             'expire': 2, 'type': int},
                                    }
 
+        #lumi.remote.b286acn01
+        wallswitch_b286acn01 = {
+          0x05F01 : 'right',
+          0x05F02 : 'left',
```

Comment:
Where did you find this device id ? it doesn't appear in the file you provide in issue